omod/src/main/webapp/resources/htmlforms/Covid19CaseManagement-EnrollmentForm.xml

```diff
@@ -0,0 +1,165 @@
+<htmlform formName="Covd-19 Case Management Enrollment"
+          formDescription="Covd-19 Case Management Enrollment"
+          formEncounterType="fa6f3ff5-b784-43fb-ab35-a08ab7dbf074"
```

Comment:
Why is the encounter type for PNC instead of "COVID-19 Case Initiation" or "COVID-19 Case Enrolment"

Reply:
This has been fixed